doc/security/p_auth_user_identity_promotion.md

```diff
@@ -69,7 +69,7 @@ The following procedure describes how to implement AMQP user identity promotion
 
 2. Compile your LoginModule and include it in a JAR file that you put into the `GATEWAY_HOME/lib` directory.
 
-  **Note:** These instructions assume the jar `com.kaazing.gateway.amqp.server.spi.jar` is added to the compile-time classpath of the login module.
+  **Note:** These instructions assume the jar `com.kaazing.gateway.service.amqp.spi.AmqpPrincipal.jar` is added to the compile-time classpath of the login module.
```

Comment:
enterprise.gateway.amqp.spi.jar is the JAR file with the AmqpPrincipal class defined.